scripts/Logistics-Pipes.zs

```diff
@@ -863,22 +905,10 @@ NEI.hide(<LogisticsPipes:item.itemUpgrade:35>);
 NEI.hide(<LogisticsPipes:item.itemUpgrade:36>);
 NEI.hide(<LogisticsPipes:item.itemModule:301>);
 
-// Hidden until added through code, 
+
 // Theese are used to connect 2 LP networks through chests and might lag, they are disabled for now
 NEI.hide(<LogisticsPipes:item.PipeItemsInvSysConnector>);
 NEI.hide(<LogisticsPipes:item.PipeItemsSystemEntranceLogistics>);
 NEI.hide(<LogisticsPipes:item.PipeItemsSystemDestinationLogistics>);
-NEI.hide(<LogisticsPipes:item.PipeItemsFirewall>);
 NEI.hide(<LogisticsPipes:item.logisticsItemCard>); // this item can't be easily added through Minetweaker

```

Comment:
You said something about lags, but does it apply to all of these four items? If it's only inv system connector, I can have a look at item card's recipe. However I'm not sure what recipe we should give it. It would most likely be a vanilla crafting table recipe though as it would be easiest to implement.

Reply:
It is the Logistics Inventory System Connector that can lag since you can chain multiple LP networks together through chests. 

The other 3 should actually be added, but the paired item card with unique nbt data makes me unable to add that through a zs script. That's why all 4 are disabled.